<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowTitle">
   <string>Генератор музыки</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_root">
    <item>
     <widget class="QGroupBox" name="groupParameters">
      <property name="title">
       <string>Параметры генерации</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_params">
       <item row="0" column="0">
        <widget class="QLabel" name="labelGenre">
         <property name="text">
          <string>Жанр:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboGenre">
         <item><property name="text"><string>Cinematic</string></property></item>
         <item><property name="text"><string>Hip-Hop</string></property></item>
         <item><property name="text"><string>EDM</string></property></item>
         <item><property name="text"><string>Rock</string></property></item>
         <item><property name="text"><string>Jazz</string></property></item>
         <item><property name="text"><string>Ambient</string></property></item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelBPM">
         <property name="text">
          <string>BPM:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spinBPM">
         <property name="minimum"><number>40</number></property>
         <property name="maximum"><number>220</number></property>
         <property name="value"><number>120</number></property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelDuration">
         <property name="text">
          <string>Длительность (с):</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="spinDuration">
         <property name="minimum"><number>10</number></property>
         <property name="maximum"><number>600</number></property>
         <property name="value"><number>60</number></property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelKey">
         <property name="text">
          <string>Тональность:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="comboKey">
         <item><property name="text"><string>Cmaj</string></property></item>
         <item><property name="text"><string>Gmaj</string></property></item>
         <item><property name="text"><string>Dmaj</string></property></item>
         <item><property name="text"><string>Amaj</string></property></item>
         <item><property name="text"><string>Emaj</string></property></item>
         <item><property name="text"><string>Bmaj</string></property></item>
         <item><property name="text"><string>F#maj</string></property></item>
         <item><property name="text"><string>C#maj</string></property></item>
         <item><property name="text"><string>Amin</string></property></item>
         <item><property name="text"><string>Emin</string></property></item>
         <item><property name="text"><string>Bmin</string></property></item>
         <item><property name="text"><string>F#min</string></property></item>
         <item><property name="text"><string>C#min</string></property></item>
         <item><property name="text"><string>G#min</string></property></item>
         <item><property name="text"><string>D#min</string></property></item>
         <item><property name="text"><string>A#min</string></property></item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupInstruments">
      <property name="title">
       <string>Инструменты</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_instr">
       <item row="0" column="0">
        <widget class="QCheckBox" name="chkDrums">
         <property name="text"><string>Ударные</string></property>
         <property name="checked"><bool>true</bool></property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="chkBass">
         <property name="text"><string>Бас</string></property>
         <property name="checked"><bool>true</bool></property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QCheckBox" name="chkPiano">
         <property name="text"><string>Фортепиано</string></property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="chkGuitar">
         <property name="text"><string>Гитара</string></property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="chkStrings">
         <property name="text"><string>Струнные</string></property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QCheckBox" name="chkPads">
         <property name="text"><string>Пэды</string></property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupActions">
      <property name="title">
       <string>Действия</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_actions">
       <item><widget class="QPushButton" name="btnGenerate"><property name="text"><string>Сгенерировать</string></property></widget></item>
       <item><widget class="QPushButton" name="btnStop"><property name="text"><string>Стоп</string></property></widget></item>
       <item><widget class="QPushButton" name="btnDownloadZip"><property name="text"><string>Скачать ZIP</string></property></widget></item>
       <item><widget class="QPushButton" name="btnEnhanceCloud"><property name="text"><string>Улучшить в облаке</string></property></widget></item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupProgress">
      <property name="title">
       <string>Прогресс и лог</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_progress">
       <item><widget class="QProgressBar" name="progressBar"><property name="value"><number>0</number></property></widget></item>
       <item><widget class="QLabel" name="labelStatus"><property name="text"><string>Готово.</string></property></widget></item>
       <item><widget class="QPlainTextEdit" name="txtLog"><property name="readOnly"><bool>true</bool></property></widget></item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
